jsapp/js/searches.es6

```diff
@@ -94,7 +93,22 @@ function SearchContext(opts={}) {
           }
         }
       };
+      var filterOutDeletedAssetFromCategorizedList = () => {
+        let list = this.state.defaultQueryCategorizedResultsLists;
+        if (list != undefined) {
+          var l = {};
+          for (var category in list) {
+            l[category] = list[category].filter(function(result){
+              return result.uid !== asset.uid;
+            });
+          }
+          let o = {};
+          o.defaultQueryCategorizedResultsLists = l;
```

Comment:
Nitpicking: what is `l` and `o`? I would be happier with longer names ;-)

Reply:
`filterOutDeletedAssetFromCategorizedList` closely emulates `filterOutDeletedAsset`, and I kept the `l` and `o` variables. They're basically temporary holders of the list data, post-filtering.

Reply:
ok :)